ext/POSIX/POSIX.xs

```diff
@@ -3407,8 +3456,50 @@ wcstombs(s, pwcs, n)
 
 int
 wctomb(s, wchar)
-	char *		s
+	SV *		s
 	wchar_t		wchar
+    PREINIT:
+            char buffer[MB_LEN_MAX];
```

Comment:
maybe initialize the buffer to `0`s

Reply:
I re-did the PR.  For one thing, I came to the conclusion that it should always use the non-r versions when not threaded.   Your comment applies to somewhat out-dated code., but there still is a 'buffer', and it could be initialized.  But I don't see why.  It's not visible outside the XS code.  But now, there is an SV that is returned that mirrors 'buffer'.  Currently that SV is set if the function returns a success RETVAL, but is untouched otherwise.  The functions man pages don't specifiy what happens to 'buffer' on failure, so I just left it untouched.  But we could invalidate the SV on failure.  I'm unsure.